index.html

```diff
@@ -17,7 +17,7 @@ <h1 class="page-heading">Posts</h1>
     {% assign post-url = {{post.url}} %}
     {% assign size = post-url | size | minus: 1 %}
 
-    <a class="post-card grow" href="{{ post.url | prepend: site.baseurl }}" style="background-image: linear-gradient(to bottom, rgba(255, 255, 255, 1), rgba(255, 255, 255, 0.6) 40%, rgba(255, 255, 255, 0) 60%), url('/images/post-covers{{ post-url | slice: 0, size }}.png');">
+    <a class="post-card grow" href="{{ post.url | prepend: site.baseurl }}" style="background-image: linear-gradient(to bottom, rgba(255, 255, 255, 1), rgba(255, 255, 255, 0.6) 40%, rgba(255, 255, 255, 0) 60%), url('/images/post-covers{{ post-url | slice: 0, size }}.jpg');">
```

Comment:
FYI, since I changed this part of the template, your future post covers will also need to end with `.jpg`